underlay/src/main/java/bio/terra/tanagra/utils/ProtobufUtils.java

```diff
@@ -4,10 +4,23 @@
 import com.google.protobuf.InvalidProtocolBufferException;
 import com.google.protobuf.Message;
 import com.google.protobuf.util.JsonFormat;
+import java.util.Base64;
 
 public final class ProtobufUtils {
+
   private ProtobufUtils() {}
 
+  @SuppressWarnings({"checkstyle:EmptyCatchBlock", "PMD.EmptyCatchBlock"})
+  public static <T extends Message.Builder> T deserializeFromJsonOrProtoBytes(
+      String serialized, T builder) {
+    try {
```

Comment:
I have similar code in the UI and I've found it annoying when debugging because of all the spurious exceptions so I was thinking about checking if the first character is a "{" (invalid for base 64) rather than trying to deserialize.

Reply:
This should swallow the exceptions, not throw or even log them, so I'm hoping I won't have the same debugging irritation. That's a nice tip though, I think it would make this code a little less ugly. I'll make the change in a separate PR.